src/Umbraco.Web.Common/Extensions/UmbracoCoreServiceCollectionExtensions.cs

```diff
@@ -296,6 +237,77 @@ public static IServiceCollection AddUmbracoCore(this IServiceCollection services
             return services;
         }
 
+        /// <summary>
+        /// Adds SqlCe support for Umbraco
+        /// </summary>
+        /// <param name="services"></param>
+        /// <returns></returns>
+        private static IServiceCollection AddUmbracoSqlCeSupport(this IServiceCollection services)
+        {
+            try
+            {
+                var binFolder = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+                if (binFolder != null)
+                {
+                    var dllPath = Path.Combine(binFolder, "Umbraco.Persistance.SqlCe.dll");
```

Comment:
Does this work cross platform?

Reply:
it "works" by throwing an exception that is ignored in the catch statement

Reply:
I've just moved this, not added it.